lib/teleterm/daemon/daemon.go

```diff
@@ -372,9 +405,21 @@ func (s *Service) Stop() {
 	s.mu.RLock()
 	defer s.mu.RUnlock()
 
+	s.Log().Info("Stopping")
+
 	for _, gateway := range s.gateways {
 		gateway.Close()
 	}
+
+	close(s.clusterEventsC)
+}
+
+func (s *Service) Log() *logrus.Entry {
+	return s.cfg.Log
+}
+
+func (s *Service) ClusterEventsC() <-chan *api.ClusterEvent {
+	return s.clusterEventsC
```

Comment:
Since I'm going to move the logic from the gRPC handler somewhere else, the request to create a stream will likely pass through `daemon.Service`. In that case, we won't even need this method. Instead, we'll just pass the channel to the struct that will handle reading from the channel and writing to the stream.